components/brave_wallet_ui/page/screens/onboarding/onboarding-success/onboarding-success.tsx

```diff
@@ -69,8 +71,11 @@ export const OnboardingSuccess = () => {
 
   // effects
   React.useEffect(() => {
+    // now that the token registry is populated, discover assets
+    discoverAssets()
+
     report(BraveWallet.OnboardingAction.Complete)
-  }, [report])
+  }, [report, discoverAssets])
```

Comment:
afaik this is not needed as mutations should be stable functions
https://github.com/reduxjs/redux-toolkit/blob/ed9fc19027a31afee2234dea54fb36c8beef46ad/packages/toolkit/src/query/react/buildHooks.ts#L1074-L1080